src/components/MailboxThread.vue

```diff
@@ -337,6 +340,7 @@ export default {
 	methods: {
 		groupEnvelopesByDate(envelopes, syncTimestamp) {
```

Comment:
This would lend itself quite well for a pure helper function that we can unit test. The same was done for mailbox sorting, for example.